portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/passwordpolicies/PasswordPolicies.testcase

```diff
@@ -536,7 +536,7 @@ definition {
 
 		User.forgotPasswordSendResetLink(userEmailAddress = "userea@liferay.com");
 
-		Pause(locator1 = "300000");
+		Pause(locator1 = "500000");
```

Comment:
Hello @melodywu,
As I remember, the pause 6000 used in test is actually 1 second, 300000 here is 5 mins then, since the test is testing for the situation of 5 mins, change to 500000 here would be too much for the test.
We'd better trying to find a new solution for the failure, thanks.

Reply:
OK, I got it . I calculate it wrong